symfony/templates/participant/details.html.twig

```diff
@@ -0,0 +1,685 @@
+{% extends 'base.html.twig' %}
+{% block title %}Participant {{ participant.id }} - {% endblock %}
+{% block body %}
+    {% import 'macros/display-text.html.twig' as macros %}
+    {% if hasNoParticipantAccess %}
+        <div class="row">
+            <div class="col-sm-8 col-sm-push-2 col-lg-6 col-lg-push-3">
+                <div class="panel panel-danger">
+                    <div class="panel-heading">
+                        <h3 class="panel-title"><i class="fa" aria-hidden="true"></i>Warning</h3>
+                    </div>
+                    <div class="panel-body">
+                        The participant you have selected is either unpaired or paired with another enrollment site. Please verify that this is the correct participant. By continuing, you acknowledge that you have a valid need to access this record. Access to this record will be audited and is subject to review. Unauthorized access to records may result in loss of access to HealthPro.
+                    </div>
+                    <div class="modal-footer text-right">
+                        <form action="" method="post">
+                            {{ form_widget(agreeForm) }}
+                            <button type="submit" class="btn btn-success">Acknowledge</button>
```

Comment:
This results in a duplicate "Acknowledge" button. To make this work, we likely need to do something like:

```
{{ form_start(agreeForm) }}
{{ form_widget(agreeForm.Acknowledge) }}
<a href="{{ path(cancelRoute) }}" class="btn btn-default">Cancel</a>
{{ form_end(agreeForm) }}

```